compiler/noirc_frontend/src/elaborator/expressions.rs

```diff
@@ -230,6 +236,41 @@ impl Elaborator<'_> {
         (HirBlockExpression { statements }, block_type)
     }
 
+    /// If the given type was declared as:
+    /// - `#[must_use = "message"]`, return `Some(Some(message))`
```

Comment:
Is this outdated from when `MustUse` used to be an `Option<Option<String>>`? Let's just link to the actual helper enum